src/app/components/colorpicker/index.html

```diff
@@ -42,11 +42,16 @@
           cmyk: {{'{'}} cyan: number, magenta: number,    yellow: number,     key: number  {{'}'}}
           hex:  string
         {{'}'}}
-        </pre></div>
+        </pre>
+      </div>
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Colorpicker input directive properties">
+    <sky-demo-page-property
+      propertyName="skyColorpickerInput">
+      Place this attribute on an input element or button element that is wrapped in a <stache-code>sky-colorpicker</stache-code> component to create the element and colorpicker. You must set this attribute to the instance of the <stache-code>sky-colorpicker</stache-code>. This directive uses <stache-code>NgModel</stache-code> to bind data.
```

Comment:
I'd reword this to mimic the wording for `skyDatepickerInput` in datepicker and `skyTimepickerInput` in timepicker: "Creates the colorpicker element and dropdown. Place this attribute on an `input` element or `button` element, and wrap the element in a `sky-colorpicker` component. The attribute must be set to the instance of the `sky-colorpicker` component."